### DISCUSSION TITLE
which python version should I use in order to install JAX?

### DISCUSSION THREAD

**Discussion by inikishev** [jax-ml/jax#19697] [Q&A]
I can't find any info on this, but it says it is incompatible with python 3.11 when I install it, but in a cryptic way, so I don't know if that is actually true

**Comment by jakevdp** [comment]
That information is in the installation metadata. The current version of JAX supports Pythn 3.9 and newer: https://github.com/google/jax/blob/ddd668e929fb12d55d37b8cd0af8097968a1cfef/setup.py#L74

If you're having difficulty on a supported Python version like 3.11, please file an issue with details and we'll do our best to help. Thanks!
